CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/parser/JSONImportFileParser.java

```diff
@@ -401,20 +402,20 @@ private void lookForChildArrays(JsonNode node, String path, int depth) throws IO
                 childNode.add(node);
                 counter++;
             }
-            // Maps newline delimited JSON to valid JSON in the format 
+            // Maps newline delimited JSON to valid JSON in the format
             // {"results": [<ndjson>]}
-            switch(counter){
-                case(0):
+            switch (counter) {
+                case (0):
                     throw new IOException(WARN_NO_VALID_LIST);
-                case(1):
+                case (1):
```

Comment:
noting they appear to already be there, the brackets can be omitted from the cases